modules/oadp-storage-class-mapping.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+[id="oadp-storage-class-mapping_{context}"]
+:_mod-docs-content-type: CONCEPT
+= Storage class mapping
+
+Storage class mapping allows you to define rules or policies specifying which storage class should be applied to different types of data. This feature automates the process of determining storage classes based on access frequency, data importance, and cost considerations. It optimizes storage efficiency and cost-effectiveness by ensuring that data is stored in the most suitable storage class for its characteristics and usage patterns.
+
+You can use the `change-storage-class-config` field to change the storage class of your data objects. This feature allows you to optimize costs and performance by moving data between different storage tiers, such as from standard to archival storage, based on your needs and access patterns.
+
+[id=storage-class-mapping-mtc_{context}]
+== Storage class mapping with MTC
+
+You can use Migration Toolkit for Containers (MTC) to migrate containers, including application data, from one OpenShift Container Platform cluster to another cluster and for storage class mapping and conversion. You can convert the storage class of a persistent volume (PV) by migrating it within the same cluster. To do so, you must create and run a migration plan in the Migration Toolkit for Containers (MTC) web console.
+
+For more information see xref:../../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-converting-storage-classes_advanced-migration-options-mtc[Migration converting storage classes].
+
+[id=storage-class-mapping-oadp_{context}]
+== Storage class mapping with OADP
+
+You can also use OpenShift API for Data Protection (OADP)  with the Velero plugin v1.1.0 and above to change the storage class of a persistent volume (PV) during restores, by configuring a storage class mapping in the config map in the Velero namespace.
+
+To employ ConfigMap with OADP, use the `change-storage-class-config` field. Ensure to change the storage class mapping based on your cloud provider.
+
+. Change the storage class mapping by running the following command:
```

Comment:
You have a procedure inside of a procedure module. I suggest breaking this content out into its own module.